neuromation/api/parser.py

```diff
@@ -64,3 +97,39 @@ def _remote_to_local_image(self, image: RemoteImage) -> LocalImage:
             self._config.username, self._config.cluster_name, self._config.registry_url
         )
         return parser.convert_to_local_image(image)
+
+    def build_env(self, env: Sequence[str], env_file: Optional[str]) -> Dict[str, str]:
```

Comment:
The same thing: the naming and functionality.
Instead of *build/split* pair, we can have `client.parse.env(env, env_file)` method that returns a pair of `Mapping[str, str]` for regular env-vars and `Mapping[str, URL]` for secret envs.

Reply:
👍

Reply:
Docker supports multiple env files (https://docs.docker.com/compose/compose-file/#env_file), so it may be better to accept a sequence of strings.

Reply:
#1711

Reply:
@serhiy-storchaka good idea